figures/mapping-histology-labels.Rmd

```diff
@@ -112,19 +160,22 @@ Check out the discussion on [#1174](https://github.com/AlexsLemonade/OpenPBTA-an
 
 ### `broad_histology`
 
+We will start by creating a palette for broad histology display values.
+Broad histology values will be recoded specifically for display in figures in later step.
+
 ```{r}
 broad_histology_df <- data.frame(
-  broad_histology = c("Benign tumor",
-                      "Diffuse astrocytic and oligodendroglial tumor",
-                      "Embryonal tumor",
-                      "Ependymal tumor",
-                      "Germ cell tumor",
-                      "Low-grade astrocytic tumor",
-                      "Meningioma",
-                      "Mesenchymal non-meningothelial tumor",
-                      "Neuronal and mixed neuronal-glial tumor",
-                      "Tumor of cranial and paraspinal nerves",
-                      "Tumors of sellar region"), 
+  broad_histology_display = c("Benign tumor",
+                              "High-grade glioma",
+                              "Embryonal tumor",
+                              "Ependymal tumor",
+                              "Germ cell tumor",
+                              "Low-grade glioma",
+                              "Meningioma",
+                              "Mesenchymal non-meningothelial tumor",
+                              "Neuronal and mixed neuronal-glial tumor",
+                              "Tumor of cranial and paraspinal nerves",
+                              "Tumors of sellar region"), 
```

Comment:
Just pointing out this one is plural; make singular match?

Reply:
This is what's in the histologies file, so if we were to do this it would mean changes later in the notebook as well, like "Low-grade astrocytic tumor" --> "Low-grade glioma"

Reply:
Tagging @jharenza  for thoughts

Reply:
This is probably just a miss - I think they maybe all used to be plural to go along with the WHO book, but then became singular as talking about one biospecimen. But, we had not been re-labeling plots to use the plural, though maybe we should (eg oncoprints have broad histology singular but the plot is of multiple tumors). I don't have a strong opinion, but if we want to be grammatically correct in plots, we probably should make `broad_histology_display` plural.

Reply:
@jaclyn-taroni and @sjspielman - just wanted to come back to this before running anymore modules/figures. I updated sample distribution plot [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/f0aa941c2e9b4f91ad33420eae3b7092e5a0fc88/figures/pdfs/fig1/panels/tumor_descriptor_proportion_panel.pdf) and have made some updates to interaction plots [here](url) (Note: legend still needs work to header the HGG and LGGs per @jashapiro's request), and so the palette as it stands looks good to me, but if we are to update `broad_histology_display` to plural, we should just do it here/now so it carries over to sample distribution and oncoprints (don't think it is anywhere else).

Reply:
What about `cancer_group_display` then? Also make that plural? To me, it's not confusing either way so long as it is consistent and recoding fewer things is going to tend to be less error prone.

Reply:
The main thing I was trying to point out with this comment was "Tumor of" vs "Tumors of." I think cancer groups with this naming scheme (tumor of XYZ, or "XYZ tumor") should all be singular or plural. Otherwise, anything is fine as long as consistent.

Reply:
I went singular because there were fewer changes required.